tests/linearizability/linearizability_test.go

```diff
@@ -133,6 +136,62 @@ func testLinearizability(ctx context.Context, t *testing.T, clus *e2e.EtcdProces
 	return operations, events
 }
 
+func patchOperationHistory(operations []porcupine.Operation, watchOps []watchEvent) []porcupine.Operation {
+	newOperations := make([]porcupine.Operation, 0, len(operations))
+	persisted := map[EtcdOperation]watchEvent{}
```

Comment:
operationId2watch ?

Reply:
Not very convinced. This function takes operations and patches them based on watch events.

Reply:
Renamed to `patchOperationBasedOnWatchEvents`